src/assets/_project/_blocks/utils/qg-load-google-api.js

```diff
@@ -13,7 +13,7 @@ import keys from '../data/qg-google-keys';
   // check if the hostname contains a specific word and assign the key accordingly
   if (window.location.hostname.search(/\bgithub\b/) !== -1) {
     googleApiKey = keys.defGoogle.docs;
-  } else if (window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b/) !== -1) {
+  } else if (window.location.hostname.search(/dev|test|localhost/) !== -1) {
```

Comment:
should we also include uat? (like recaptcha key swap?)

Reply:
made the change will send a pull request for the release 4.0.2 patch release which I just did

Reply:
UAT env we usually test with PROD set up to make sure it is working before PROD deployment